docs/docs/index.md

```diff
@@ -1,96 +1,142 @@
 # Overview
 
-Dataview is a live index and query engine over your knowledge base. You can associate *data* (like tags, dates,
-snippets, numbers, and so on) with your markdown pages, and then *query* (like filter, sort, transform) this data. This
-is a simple but powerful idea:
+Dataview is a live index and query engine over your personal knowledge base. You can **add metadata** to your notes and **query** them to list, filter, sort or group them - dataview keeps your queries always up to date and makes data aggregation a breeze.
+You could
 
-- Track sleep schedules and habits by recording them in daily notes, and automatically create weekly tables of your
-  sleep schedule.
+- Track your sleep by recording it in daily notes, and automatically create weekly tables of your sleep schedule.
 - Automatically collect links to books in your notes, and render them all sorted by rating.
-- Automatically collect pages annotated with a given date, showing them in your daily note or elsewhere.
+- Automatically collect pages associated with today's date and show them in your daily note.
 - Find pages with no tags for follow-up, or show pretty views of specifically-tagged pages.
-- Create dynamic views which show upcoming birthdays or events, annotated with notes.
+- Create dynamic views which show upcoming birthdays or events recorded in your notes
 
-Dataview is highly generic and high performance, scaling up to hundreds of thousands of annotated notes without
-issue. If the built in [query language](query/queries/) is insufficient for your purpose, you can run arbitrary
-JavaScript against the [dataview API](api/intro/).
+and many, many things more.
 
-## Basic Usage
+Dataview is highly generic and high performance, scaling up to hundreds of thousands of annotated notes without issue. 
+**Dataview gives you a fast way to search, display and operate on indexed data**. 
 
-Dataview has two major components: *annotation* and *querying*. Each operates largely independently and are described below.
+If the built in [query language](query/queries/) is insufficient for your purpose, you can run arbitrary
+JavaScript against the [dataview API](api/intro/) and build whatever utility you might need yourself, right in your notes.
 
-#### Annotation
+!!! info "Dataview is about displaying, not editing"
+    Dataview is meant for displaying and calculating data. It is no utility to edit data in your source files and will keep your notes always untouched (... except you're checking a [Task](queries/query-types.md#task-queries) through Dataview.)
```

Comment:
typo: "except *if* you're checking"

Reply:
Thanks for the pointer. I fixed the missing if and also added some more interlinking to other pages. Would be great if you could have another look :)